### PR TITLE
Added a login UI and added feature to check for user credentials and …

### PR DESCRIPTION
**Pull request by dionwu110**
Updated database, added session table.
Added login features

### CODE REVIEW

File: login.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+.form-control {
+  border-radius: 10px;
+}
+
+.alignment {
+  text-align: center;
+}
+
+p.invalidUser, p.invalidPass{display:none;}
```
Comment (danqo) [inline login.css]: is this normal style? how about:
`p.invalidUser, p.invalidPass { display: none; } `

Jeffrey to comment

**Comment by danqo** [comment]
also can you upload a screenshot of testing done

File: session.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        <li class="page-item"><a class="page-link" href="#">2</a></li>
+        <li class="page-item"><a class="page-link" href="#">3</a></li>
+        <li class="page-item"><a class="page-link" href="#">Next</a></li>
+      </ul>
+    </div>
+    
+    <!-- Search Function -->
+    <script>
+    $(document).ready(function(){
+      $("#searchInput").on("keyup", function(){
```
Comment (lydarren) [inline session.html]: spacing after function()

File: login.css (hunk 1 above)
Comment (jtam360) [reply login.css]: Yeah it should have spaces in the brackets.

**Comment by dionwu110** [comment]
Tests:
login page

loginvalidation
Login form is now centered based on window size, checks for valid login credentials

database
Goes to database when inputted valid login credentials

File: session.html (hunk 2 above)
Comment (lydarren) [inline session.html]: spacing

**Pull request merged**